libs/WordPressAnnotations/build.gradle

```diff
@@ -1,24 +1,10 @@
 apply plugin: 'kotlin'
 
-dependencies {
-    ext.kotlinVersion = '1.4.10'
-
-    implementation "org.jetbrains.kotlin:kotlin-stdlib:$kotlinVersion"
-}
-
-sourceCompatibility = "7"
-targetCompatibility = "7"
-
 repositories {
     mavenCentral()
 }
-compileKotlin {
-    kotlinOptions {
-        jvmTarget = "1.8"
```

Comment:
Unsure about removing the explicit jvm target here… wouldn't that risk of creating discrepancies between people using different JVMs on their machine (if not everybody updates to the latest JVM all at once etc)?

Reply:
That's a fair question. I've removed this in preparation for the [Android Gradle Plugin 4.2.0 upgrade which uses Java 8 by default](https://developer.android.com/studio/releases/gradle-plugin#java-8-default). However, maybe doing this as part of this PR wasn't the correct choice, so I've reverted jvm & Kotlin related changes in a3f27c2d7b31af4d2ee52628b67f35a51bab5f63 & 33a892ee9e55d0e25408d237e0e3be709b7a29e3 and will handle it in a separate PR.